From 51a2a37e1944210a4556df5f4468f705f905fe72 Mon Sep 17 00:00:00 2001
From: Thierry Reding <thierry.reding@avionic-design.de>
Date: Wed, 15 Aug 2012 07:48:04 +0200
Subject: [PATCH] Fix out-of-tree build

For out of tree builds, the top-level source directory is not added to
the include path automatically, so this needs to be done explicitly.

Signed-off-by: Thierry Reding <thierry.reding@avionic-design.de>
---
 test/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/Makefile.am b/test/Makefile.am
index 0cdd859..d66e2ac 100644
--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -1,7 +1,7 @@
 noinst_PROGRAMS = mtdev-mapgen mtdev-matching mtdev-kernel
 bin_PROGRAMS = mtdev-test
 
-INCLUDES=-I$(top_srcdir)/include/
+INCLUDES=-I$(top_srcdir)/include/ -I$(top_srcdir)
 
 mtdev_mapgen_SOURCES = mtdev-mapgen.c
 
-- 
1.7.11.4

